PWGLF/TableProducer/cascadebuilder.cxx

```diff
@@ -90,7 +90,7 @@ struct cascadebuilder {
   Configurable<bool> tpcrefit{"tpcrefit", true, "demand TPC refit"};
   Configurable<double> v0radius{"v0radius", 0.9, "v0radius"};
 
-  void process(aod::MatchedV0Cascades const& MatchedV0Cascades, aod::V0Datas const&, aod::Cascades const&, aod::Collisions const&, soa::Join<aod::FullTracks, aod::TracksExtended> const&)
+  void process(aod::MatchedV0Cascades const& MatchedV0Cascades, aod::V0Datas const&, aod::Cascades const&, aod::Collisions const&, soa::Join<aod::FullTracks, aod::TracksCov, aod::TracksExtended> const&)
```

Comment:
Silly but simple comment, if I may ... let's just use the FullTracksExt defined at the top and subscribe to that (or remove it just to clean it up). Is that OK, @aimeric-landou? Thanks!